### ISSUE TITLE
Create a quick user facing doc on data classification 

### ISSUE THREAD

**Issue by dirkpetersen** [FredHutch/wiki#60]
We are using terms such as  "protected data"  in the wiki. The hutch has now a data classification standard 

https://centernet.fredhutch.org/cn/u/hdc/_jcr_content/leftParsys/download_0/file.res/Fred-Hutch-Information-Classification-and-Handling-Standard-FINAL.pdf

Can we motivate HDC/ISO to write a quick summary for the Wiki

**Comment by vortexing** [comment]
Sounds great.  I think that a document about this issue would be great to add to this existing page:
http://sciwiki.fredhutch.org/generation/human_privacySecurity/

That is where we have started putting information about how to think about privacy and security when human data is included.

**Comment by vortexing** [comment]
@dirkpetersen do you have any updates on getting someone to contribute to this?  Have you talked to @sgglick about it?  Is that something more appropriate for someone else?

**Comment by sgglick** [comment]
Dirk did talk to me and I did create one and sent it to ISO to review a few days ago.  I have attached a chart you can use (ISO has not formally approved, so we may need to revise once they do).  The main issue to be aware of is that standard demands functions not currently available to  most at FH: encryption at rest and encrypted email.  There is a notation about this in the chart; it was decided to postpone the effectiveness date of the encryption requirement.

I have also provided a PII vs PHI chart and the 19 elements of de-identification (within same excel file).  Finally I have posted an in-depth de-identification guide for structured, NON-Genomic data.

Let me know…
Susan

Research Compliance Specialist / Hutch Data Commonwealth / 206.667.2507 / sgglick@fredhutch.org<mailto:sgglick@fredhutch.org> / Fred Hutch / Cures Start Here

**Comment by vortexing** [comment]
Will follow up with Susan via email.

**Comment by sgglick** [comment]
Dirk did talk to me and I did create one and sent it to ISO to review a few days ago.  I have attached a chart you can use (ISO has not formally approved, so we may need to revise once they do).  The main issue to be aware of is that standard demands functions not currently available to  most at FH: encryption at rest and encrypted email.  There is a notation about this in the chart; it was decided to postpone the effectiveness date of the encryption requirement.

I have also provided a PII vs PHI chart and the 19 elements of de-identification (within same excel file).  Finally I have posted an in-depth de-identification guide for structured, NON-Genomic data.

Let me know...
Susan

Research Compliance Specialist / Hutch Data Commonwealth / 206.667.2507 / sgglick@fredhutch.org<mailto:sgglick@fredhutch.org> / Fred Hutch / Cures Start Here

**Comment by vortexing** [comment]
I'm not sure after the various discussions that have happened, what exactly is the sentiment behind what this issue was about.  What portion of what material makes sense to put up on the Wiki?

**Comment by sgglick** [comment]
Saw some discussion with Dirk on this.  At this point in time, the individuals I know in ISO believe we need some minor revisions to the data classification standard.  If we post the easy to read chart I sent to you, we would need to amend that chart  after the revisions to the standard are approved.

So ultimately, there was no conclusive opinion really on the chart from ISO, etc. The standard is on the centernet available to all at this moment. It has not be “evangelized”.  I think this year’s annual security training may include it…not sure.  The graphic/chart I sent you replicates the standard.

Honestly, if you want to post it as it is, I will take any backlash, if any occurs.  We will need to update once the revisions to the data classification standard is approved, sometime in the future.

Does this help?
Susan



Research Compliance Specialist / Hutch Data Commonwealth / 206.667.2507 / sgglick@fredhutch.org<mailto:sgglick@fredhutch.org> / Fred Hutch / Cures Start Here

**Comment by vortexing** [comment]
@sgglick Can I put a markdown in _gendemos to hold a summary of this content?  Like a markdown table of PII vs PHI and also the 19 identifiers and a summary of the step by step guide to de-identify genomic data sets?  Then I can link to that from the content to make it shorter.  OR possibly integrate it into the content and add links to some future site you have in CenterNet for this more detailed content?

**Comment by vortexing** [comment]
done
